react/features/remote-video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -274,7 +280,8 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
  *     _disableKick: boolean,
  *     _disableRemoteMute: boolean,
  *     _menuPosition: string,
- *     _remoteControlState: number
+ *     _remoteControlState: number,
```

Comment:
Can you drop this entire comment part and say it returns `{Props}` ? It makes no sense to duplicate that information, so we have been slowly removing them as we modify code.